docs/serverless/08-05-set-external-docker-registry.md

```diff
@@ -0,0 +1,296 @@
+
+    ```bash
+    export SP_APP_ID=$(az ad sp show --id http://${AZ_SERVICE_PRINCIPAL_NAME} --query appId --output tsv)
+    export SP_PASSWORD=$(az ad sp show --id http://${AZ_SERVICE_PRINCIPAL_NAME} --query password --output tsv)
+    az role assignment create --assignee ${SP_APP_ID} --scope ${AZ_REGISTRY_ID} --role ${ROLE}
+    ```
+
+  </details>
+</div>
+
+### Override Serverless configuration
+
+Apply the following Secret with an override to a cluster or Minikube. Run:
+
+<div tabs name="override" group="external-docker-registry">
+  <details>
+  <summary label="docker-hub">
+  Docker Hub
+  </summary>
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: serverless-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: serverless
+    kyma-project.io/installation: ""
+data:
+  dockerRegistry.enableInternal: $(echo "false" | base64)
+  dockerRegistry.username: $(echo "${USERNAME}" | base64)
+  dockerRegistry.password: $(echo "${PASSWORD}" | base64)
+  dockerRegistry.serverAddress: $(echo "${SERVER_ADDRESS}" | base64)
+  dockerRegistry.registryAddress: $(echo "${REGISTRY_ADDRESS}" | base64)
+EOF
+```
+
+  </details>
+  <details>
+  <summary label="gcr">
+  GCR
+  </summary>
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: serverless-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: serverless
+    kyma-project.io/installation: ""
+data:
+  dockerRegistry.enableInternal: $(echo "false" | base64)
+  dockerRegistry.username: $(echo "_json_key" | base64)
+  dockerRegistry.password: "${GCS_KEY_JSON}"
+  dockerRegistry.serverAddress: $(echo "${SERVER_ADDRESS}" | base64)
+  dockerRegistry.registryAddress: $(echo "${SERVER_ADDRESS}/${PROJECT}" | base64)
+EOF
+```
+
+  </details>
+  <details>
+  <summary label="acr">
+  ACR
+  </summary>
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: serverless-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: serverless
+    kyma-project.io/installation: ""
+data:
+  dockerRegistry.enableInternal: $(echo "false" | base64)
+  dockerRegistry.username: $(echo "${SP_APP_ID}" | base64)
+  dockerRegistry.password: $(echo "${SP_PASSWORD}" | base64)
+  dockerRegistry.serverAddress: $(echo "${AZ_REGISTRY_NAME}.${SERVER_ADDRESS}" | base64)
+  dockerRegistry.registryAddress: $(echo "${AZ_REGISTRY_NAME}.${SERVER_ADDRESS}" | base64)
+EOF
+```
+
+  </details>
+</div>
+
+> **CAUTION:** If you want to set an external Docker registry before you install Kyma, manually add the Secret to the `installer-config-local.yaml.tpl` (local installation) or `installer-config-production.yaml.tpl` (production installation) template located under the `installation/resources` subfolder before you run the installation script.
```

Comment:
No, you shouldn't modify `*.tpl` files, simply apply secret with an override to the cluster before triggering the installation.